script/build.js

```diff
@@ -195,16 +195,18 @@ if (options.watch) {
     // Check to see if we have a proxy config file
     const api = require('../config/config.proxy.js').api;
     devServerConfig.proxy = {
-      '/api/*': {
-        target: `https://${api.host}${api.path}`,
+     '/rx-api/*': {
+        target: `https://${api.host}/`,
         auth: api.auth,
-        rewrite: function (req) {
-          req.url = req.url.replace(/^\/api/, '');
+        secure: true,
+        changeOrigin: true,
+        pathRewrite: function(path, req){
```

Comment:
Yep, `http-proxy-middleware`.  We're kind of on the bleeding edge here, since we're piling a lot of functionality into webpack-dev-server.  I expect this to get simpler over time as we start to consolidate various projects.